src/modules/UI/components/WalletListRow/WalletListRow.ui.js

```diff
@@ -8,7 +8,7 @@ import { TouchableHighlight, View } from 'react-native'
 import slowlog from 'react-native-slowlog'
 
 import { intl } from '../../../../locales/intl.js'
-import type { CurrencyConverter, CustomTokenInfo, GuiDenomination, GuiWallet } from '../../../../types.js'
+import type { CustomTokenInfo, GuiDenomination, GuiWallet } from '../../../../types.js'
 import {
```

Comment:
Now we will be removing ALL occurrences of `CurrencyConverter` in the GUI except for where we are building exchange rates.